src/shipit_uplift/shipit_uplift/coverage.py

```diff
@@ -6,15 +6,22 @@
 from abc import ABC, abstractmethod
 import json
 from functools import lru_cache
+from cachetools import LRUCache
 import requests
 
+from shipit_uplift import secrets
 
-@lru_cache(maxsize=128)
+
+@lru_cache(maxsize=2048)
```

Comment:
Why the increase ?

Reply:
The larger cache should speed up more requests.